config/strategies/googleStrategy.js

```diff
@@ -0,0 +1,36 @@
+import passport from 'passport';
+import { OAuth2Strategy as GoogleStrategy } from 'passport-google-oauth';
+import Users from '../../controllers/user';
```

Comment:
Could you import the `Users` controller directly from the controllers directory?